src/python/constants.py

```diff
@@ -31,6 +31,12 @@
 WIKI_MEDIUM_DOCS_LINE_FILE = '%s/data/enwiki-20120502-lines-1k.txt' % BASE_DIR
 WIKI_MEDIUM_DOCS_COUNT = 33332620
 
+# Word vectors downloaded from http://nlp.stanford.edu/data/glove.6B.zip
+# Licensed under Public Domain (http://www.opendatacommons.org/licenses/pddl/1.0/);
+# see https://nlp.stanford.edu/projects/glove/
+# Thanks to Jeffrey Pennington, Richard Socher, and Christopher D. Manning.
```

Comment:
Thanks for including this metadata about our vectors source!  I am happy it is a generous license :)

Reply:
Hmm maybe also add how big these are?  Looks like ~823 MB zip'd and ~2.1 GB uncompressed.